test/ComposerRequireCheckerTest/DefinedExtensionsResolver/DefinedExtensionsResolverTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace ComposerRequireCheckerTest\DefinedExtensionsResolver;
+
+use ComposerRequireChecker\DefinedExtensionsResolver\DefinedExtensionsResolver;
+use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
+use PHPUnit\Framework\TestCase;
+
+class DefinedExtensionsResolverTest extends TestCase
```

Comment:
Please add `/** @covers \ComposerRequireChecker\DefinedExtensionsResolver\DefinedExtensionsResolver */` to prevent coverage leaks